src/main/java/org/myrobotlab/swing/ProgramABGui.java

```diff
@@ -40,9 +40,9 @@
   static final String START_SESSION_LABEL = "Start Session";
   // TODO: make this auto-resize when added to gui..
   private JTextField text = new JTextField("", 30);
-  private JTextArea response = new JTextArea("BOT Response :");
+  private JEditorPane response = new JEditorPane("text/html", "BOT Response :");

```

Comment:
Just a quick note: This compontent doesn't allow it's content to be modified, correct? Can't test right now.
Would otherwise need a readonly flag imo.

Reply:
Agreed, this should be immutable .. you should be able to edit this panel.